pyproject.toml

```
@@ -0,0 +1,17 @@
+[tool.ruff]
+target-version = "py38"
+line-length = 120
+# Enable Pyflakes `E` and `F` codes by default.
+select = [
+#    "E",
+#    "W", # see: https://pypi.org/project/pycodestyle
+    "F", # see: https://pypi.org/project/pyflakes
+#    "I", #see: https://pypi.org/project/isort/
+#    "D", # see: https://pypi.org/project/pydocstyle
+#    "N", # see: https://pypi.org/project/pep8-naming
+#    "S", # see: https://pypi.org/project/flake8-bandit
+]
+
+
+[tool.mypy]
+ignore_missing_imports = "True"
```

Not sure if it makes a difference
```suggestion
ignore_missing_imports = true
```

---

That may not be valid yaml syntax

---

But it's TOML. You can see it should be lowercase [here](https://mypy.readthedocs.io/en/stable/config_file.html#example-pyproject-toml).